scripts/sync.js

```diff
@@ -35,61 +35,20 @@ projectNames.forEach((name) => {
 async function RunCommand () {
   program.parse(process.argv)
   config.update(program)
-  
+
   if (program.init || program.submodule_sync) {
     progressLog('Updating submodules...')
     util.submoduleSync()
     progressLog('Done updating submodules...')
   }
-  
+
   if (program.init) {
     util.buildGClientConfig()
   }
-  
-  if (program.init) {
-    progressLog(`Syncing Gclient (with reset)`)
-    util.gclientSync(true)
-  }
-  
-  progressLog('Updating project repositories...')
-  const alwaysReset = program.init ? true : false
-  if (alwaysReset) {
-    console.log(chalk.italic('A git reset will be performed for all repositories because the "init" param was specified'))
-  }
-  let wasSomeDepUpdated = alwaysReset ? true : false
-  
-  const projectUpdateStatus = {}
-  await Promise.all(
-    projectNames.map(async (name) => {
-      let project = config.projects[name]
-      if (alwaysReset || program.all || program[project.arg_name + '_ref']) {
-        projectUpdateStatus[name] = {
-          name,
-          phase: 'fetching...',
-          ref: project.ref
-        }
-        logUpdateStatus(projectUpdateStatus)
-        await util.fetch(project.dir)
```

Comment:
This is a definite change in behavior that breaks the differential patching introduced with https://github.com/brave/brave-browser/pull/4993

Could you check that PR out and see if it still makes sense? on a higher power machine, doing a sync (and having the entire branch reset) isn't horrible. But on a laptop, the reduced build time when jumping between projects is appreciated

Reply:
There is a slight change in behavior as discussed in brave-core slack, but this does not break differential patching because it only resets the chromium branch on `init`, not on `sync`

Reply:
and this should be much faster for any machine because it only runs a single gclient sync and no other fetches, etc..